model/edx.go

```diff
@@ -129,3 +128,16 @@ func (e *EdxLMS) GetChannel(launchData map[string]string) (string, *AppError) {
 
 	return channelSlug, nil
 }
+
+func (e *EdxLMS) SyncUser(user *User, launchData map[string]string) *User {
```

Comment:
Should we handle the case where `user` is `nil`?

Reply:
No, as discussed.